docs/installation/linux/ubuntulinux.md

```diff
@@ -62,7 +62,7 @@ packages from the new repository:
 
 4. Add the new `GPG` key.
 
-        $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+        $ sudo apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
 
```

Comment:
Note that `p80` is not a single server, but a pool as well (https://sks-keyservers.net/overview-of-pools.php);

> p80.pool.sks-keyservers.net
> This is a pool containing only servers available on port 80 (needs to be used as hkp://p80.pool.sks-keyservers.net:80)

Perhaps should use `ha.pool.sks-keyservers.net`, like we do in the [install script](https://github.com/docker/docker/blob/996138bf8e6acde9a937755ef81459f7d7566fc9/hack/install.sh#L31-L35);

> ha.pool.sks-keyservers.net

This is a high-availibility subset of the pool that require all servers to be identified as a clustered setup (marked with blue indicator for reverse proxy in the [status pages](https://sks-keyservers.net/status/))